### PR TITLE
No siloless collapse until 13:00

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

The hive can't collapse because of silo less before 13:00.
Also, if the xenos didn't put any silo at 13:00, launch the timer anyway.

Change the message for siloless starting timer, for more clarity

## Why It's Good For The Game

No siloless collapse before 13:00 is a request from people who think it will reduce massive unga ball rush. I disagree, I think this will do nothing but i don't see any reason not to add it anyway. 

Has the side effect of making siloless timer ticks even if you never put any silo in the first place. I don't think it's a big deal:
Silo-less strategy are a thing, and i see no problem with that, it's just emergent gameplay. If anything, this proves that pools have issue. 
On the other hand, if you've never put a silo by 13:00, that means you are probably just delaying. Silo less is a valid strategy to make a unga ball running around without any goal, but you shouldn't be able to do it for the whole duration of the round, cause the lack of respawn will mean you are just playing a losing game. 

## Changelog
:cl:
balance: No more siloless collapse timer before 13:00. If you didn't put a silo before that time, the timer will still show up even if you never lost any silos
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -3. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by Chaznoodles** [comment]
image

Local contributor now trying to force xenos to use a feature they near-universally dislike, whilst making the marine rush have less of an impact. Think about the knock-on effects of this in conjunction with the other pool PR.

**Comment by BraveMole** [comment]
> image
> 
> Local contributor now trying to force xenos to use a feature they near-universally dislike, whilst making the marine rush have less of an impact. Think about the knock-on effects of this in conjunction with the other pool PR.

This time you're even quoting yourself! That's amazing

**Comment by Chaznoodles** [comment]
All your doing is reinforcing to the playerbase that you don't take feedback, by acting snarky and on a high horse.

**Comment by BraveMole** [comment]
> All your doing is reinforcing to the playerbase that you don't take feedback, by acting snarky and on a high horse over multiple PRs.

You're really expecting me to listen to you? Yeah, not gonna happen.

**Comment by Losenis** [comment]
>Near-universally
Astroturfing doesn't work, despite all the issues I still prefer pools to silos on both sides. Beats getting flung and immediately permakilled, or the ungas who run off on their own actually costing you the game due to giving away free respawns. Or having to walk all the way back to silos as beno because it's either that or wasting away making a new one.

**Comment by TiviPlus** [comment]
@Chaznoodles @BraveMole The most important part of constructive feedback is the constructive part

File: code/modules/mob/living/carbon/xenomorph/hive_datum.dm [hunk 1]
```diff
@@ -1148,5 +1150,5 @@ to_chat will check for valid clients itself already so no need to double check f
 	if(D?.siloless_hive_timer)
 		return
 
-	xeno_message("<span class='xenoannounce'>A sudden tremor ripples through the hive... the last silo was destroyed! The hive will collapse if nothing is done</span>", 3, TRUE)
+	xeno_message("<span class='xenoannounce'>We don't have any silos! The hive will collapse if nothing is done</span>", 3, TRUE)
```
Comment (TiviPlus) [inline code/modules/mob/living/carbon/xenomorph/hive_datum.dm:1153]: Why?

Comment (BraveMole) [reply code/modules/mob/living/carbon/xenomorph/hive_datum.dm:1153]: If you lose all silos at 12 : 50 for exemple, you won't get a message before 13 : 00. 
Same can be said if you never put any silo before 13 : 00 , so this message is more relevant

Comment (TiviPlus) [reply code/modules/mob/living/carbon/xenomorph/hive_datum.dm:1153]: Why did you change the message + not documented it in the PR?

Comment (BraveMole) [reply code/modules/mob/living/carbon/xenomorph/hive_datum.dm:1153]: Cause i thought this was absolutly not important at all and still do

Comment (TiviPlus) [reply code/modules/mob/living/carbon/xenomorph/hive_datum.dm:1153]: 'k

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**